components/sup/src/manager/mod.rs

```diff
@@ -662,9 +665,13 @@ impl Manager {
         outputln!("Starting ctl-gateway on {}", &ctl_listen_addr);
         ctl_gateway::server::run(ctl_listen_addr, ctl_secret_key, ctl_tx);
         debug!("ctl-gateway started");
-        outputln!("Starting http-gateway on {}", &http_listen_addr);
-        http_gateway::Server::new(self.fs_cfg.clone(), http_listen_addr).start()?;
-        debug!("http-gateway started");
+
+        if !self.http_disable {
```

Comment:
Might be nice to do something like
```rust
        if self.http_disable {
            info!("http-gateway disabled");
        else {
            …
```
This adds some useful information and avoids the `!disable`; I find double-negations a little bit hard to parse.

Reply:
Yeah, fair point.  I'll add that.